2018-02-22T21:18:43.927Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Object with ID 'undefined' in collection with ID 'Participant:org.feri.model.Prodajalec' does not exist)
    at /home/hlf/placilo-po-povzetju-2/angular-app/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-02-22T21:18:43.934Z ERROR   HLFConnection           :queryChainCode()         {"message":"Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID 'undefined' in collection with ID 'Participant:org.feri.model.Prodajalec' does not exist)","stack":"Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID 'undefined' in collection with ID 'Participant:org.feri.model.Prodajalec' does not exist)\n    at channel.queryByChaincode.then.catch (/home/hlf/placilo-po-povzetju-2/angular-app/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:809:34)\n    at <anonymous>"}$
2018-02-22T21:30:43.188Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Object with ID 'janez.novak@email.si
 ' in collection with ID 'Participant:org.feri.model.Kupec' does not exist)
    at /home/hlf/placilo-po-povzetju-2/angular-app/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-02-22T21:30:43.189Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Object with ID 'janez.novak@email.si\n ' in collection with ID 'Participant:org.feri.model.Kupec' does not exist)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Object with ID 'janez.novak@email.si\n ' in collection with ID 'Participant:org.feri.model.Kupec' does not exist)\n\n    at _initializeChannel.then.then.then.then.catch (/home/hlf/placilo-po-povzetju-2/angular-app/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-02-22T21:49:29.916Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'POSILJKA_001' as the object already exists)
    at /home/hlf/placilo-po-povzetju-2/angular-app/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-02-22T21:49:29.916Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'POSILJKA_001' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'POSILJKA_001' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/hlf/placilo-po-povzetju-2/angular-app/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-02-22T21:50:39.569Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'POSILJKA_001' as the object already exists)
    at /home/hlf/placilo-po-povzetju-2/angular-app/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-02-22T21:50:39.570Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'POSILJKA_001' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'POSILJKA_001' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/hlf/placilo-po-povzetju-2/angular-app/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-02-22T21:52:07.499Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Object with ID 'janez.novak@email.si
 ' in collection with ID 'Participant:org.feri.model.Kupec' does not exist)
    at /home/hlf/placilo-po-povzetju-2/angular-app/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-02-22T21:52:07.499Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Object with ID 'janez.novak@email.si\n ' in collection with ID 'Participant:org.feri.model.Kupec' does not exist)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Object with ID 'janez.novak@email.si\n ' in collection with ID 'Participant:org.feri.model.Kupec' does not exist)\n\n    at _initializeChannel.then.then.then.then.catch (/home/hlf/placilo-po-povzetju-2/angular-app/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
